utils/check-branches.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+
+# this script is checking all projects, so that they contain a branch, that is passed as parameter.
```

Comment:
please add this to the GHA for announcing the release is done so we can verify all branches exist before the announcement is made, or else throw an error reporting that some branches do not yet exist.